samples/Bluetooth/BluetoothLESample1/Program.cs

```diff
@@ -0,0 +1,240 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+using nanoFramework.Device.Bluetooth;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+
+/// <summary>
+/// Bluetooth Sample 1 is a custom service which shows the use of:
+/// 
+/// - Read static value 
+/// - Read a dynamic value using an event 
+/// - Notifying clients of changed value
+/// 
+/// You will be able to connect to the service and read values or subscribe to Notified every 10 seconds.
+/// Suitable Phone apps: "LightBlue" or "nRF Connect"
+/// </summary>
+namespace BluetoothLESample1
+{
+    public class Program
+    {
+        static GattLocalCharacteristic _readCharacteristic;
```

Comment:
I know it's a sample but always better to follow the same rules as for the rest of the class. So having the visibility explicit is always better. in this case private static for all of them